src/modules/NewDream/NewDream.js

```diff
@@ -102,8 +118,9 @@ const NewDream = (props) => {
             }}
           ></TextField>
           {/* {error && <h6>{error}</h6>} */}
-          <Button variant="contained" color="primary" onClick={submitDream}>
+          <Button variant="contained" color="primary" disabled={disabled} onClick={submitDream}>
             Add
```

Comment:
possible to change to 
`{!loading && 'Add'}` ?

Reply:
Altered